src/index.js

```diff
@@ -52,18 +50,44 @@ import createTreeMapTests from './project-tests/tree-map-tests';
 import createRandomQuoteMachineTests from './project-tests/quote-machine-tests';
 import createHeatMapTests from './project-tests/heat-map-tests';
 
+chai.config.includeStack = true;
+
 export const assert = chai.assert;
 
 let projectNameLocal = false;
+// filter out unnecessary stack trace in Mocha reporter
+const filterStack = stackTraceFilter();
+// Wrapper for shadow DOM
+const testDiv = document.createElement('div');
+testDiv.setAttribute('id', 'fcc_test_suite_wrapper');
+document.body.appendChild(testDiv);
+// Using a shadow DOM, the fCC css won't interfere with student css
+// A fallback div is provided.
```

Comment:
Can we get rid of the fallback div? We are only supporting recent versions of Chrome for now.